src/main/java/frc/robot/Robot.java

```diff
@@ -202,16 +210,28 @@ public CommandBase shootAtSpeed(final int ballAmount, double shooterSpeed) {
         return cmd;
     }
 
+    public CommandBase afterMatchPit() {
+        final CommandBase cmd = new SequentialCommandGroup(lift.resetLift(), intake.retractIntake());
```

Comment:
Pssst. We can probably condense these to one line with `return sequential("Name", foo, bar);`